ballista/rust/scheduler/src/state/execution_graph.rs

```diff
@@ -15,61 +15,40 @@
 // specific language governing permissions and limitations
```

Comment:
Should this entire file move to src/state/execution_graph/mod.rs ?

Reply:
I think `git mv` should keep it intact? Definitely if it's a single commit and we fast-forward (vs squash) merge it will.